theme/aplus/static/default.css

```diff
@@ -368,10 +368,12 @@ abbr, acronym {
     border-bottom: dotted 1px;
     cursor: help;
 }
-
+  
 /* -- code displays --------------------------------------------------------- */
-
+  
 pre {
+    color: inherit;
+    background-color: inherit;
```

Comment:
So, for the "Fix conflict between the highlight.js styles and Bootstrap" these are the only two required lines?

Reply:
Yes, the issue was focus on background color and font color of the highlighted text.